openpype/scripts/ocio_wrapper.py

```diff
@@ -96,11 +96,41 @@ def _get_colorspace_data(config_path):
 
     config = ocio.Config().CreateFromFile(str(config_path))
 
-    return {
-        c.getName(): c.getFamily()
-        for c in config.getColorSpaces()
+    colorspace_data = {
+        color.getName(): {
+            "type": "colorspace",
+            "family": color.getFamily(),
+            "categories": list(color.getCategories()),
+            "aliases": list(color.getAliases()),
+            "equalitygroup": color.getEqualityGroup(),
+        }
+        for color in config.getColorSpaces()
     }
 
+    # add looks
+    looks = config.getLooks()
+    if looks:
+        colorspace_data.update({
+            look.getName(): {
+                "type": "look",
+                "process_space": look.getProcessSpace()
+            }
+            for look in looks
+        })
+
+    # add roles
+    roles = config.getRoles()
+    if roles:
+        colorspace_data.update({
+            role[0]: {
+                "type": "role",
+                "colorspace": role[1]
+            }
+            for role in roles
+        })
+
+    return colorspace_data
```

Comment:
Isn't it technically possible for an OCIO config to define names for looks, roles and colorspaces that clash. E.g. the OCIO config can technically hold a role `srgb` and a look `srgb` and a colorspace `srgb`. By flattening them into a dict here you're essentially losing data in that case. (Plus you're making it harder later to just query, 'give me all roles' from that data).

Why note return:
```{
  "colorspaces": {},
  "roles": {},
  "looks": {}
}
```
Which can always be expanded later with even more keys if somehow OCIO starts adding features.

Reply:
Yes lets do it this way.